CRM/Export/BAO/Export.php

```diff
@@ -1311,12 +1311,9 @@ public static function sqlColumnDefn($query, &$sqlColumns, $field) {
           break;
 
         case CRM_Utils_Type::T_STRING:
-          if (isset($query->_fields[$field]['maxlength'])) {
-            $sqlColumns[$fieldName] = "$fieldName varchar({$query->_fields[$field]['maxlength']})";
-          }
-          else {
-            $sqlColumns[$fieldName] = "$fieldName varchar(255)";
-          }
+          // May be option labels, which could be up to 255 characters
+          $length = max(512, CRM_Utils_Array::value('maxlength', $query->_fields[$field]));
+          $sqlColumns[$fieldName] = "$fieldName varchar($length)";
```

Comment:
@agh1 The test seems to be failing because we are ignoring the length of all the regular columns (eg. first name, last name, etc) and using varchar 512 instead.

IMO we shouldn't do any changes here since the maxlength for option_value label is set at L#1376.